test/test.py

```diff
@@ -122,3 +182,29 @@ def test_simulation(docker_client, grape_image, reference_directory, working_dir
     assert metrics['7']['Recall'] > 0 and metrics['1']['Precision'] > 0.9
     assert metrics['8']['Recall'] > 0 and metrics['1']['Precision'] > 0.9
     assert metrics['9']['Recall'] > 0 and metrics['1']['Precision'] > 0.9
+
+
+@pytest.mark.parametrize('working_directory,preprocess_command', real_data_list, indirect=True)
+def test_real_data(docker_client, grape_image, reference_directory,
```

Comment:
I suggest to split this test into two different tests: one for `khazar` and one for `aadr`.